test/runtests.jl

```diff
@@ -23,11 +27,57 @@ using AtomsBase
     @test isnothing(ccread("./data/invalid_file.txt"))
     @test isnothing(ccread("./data/"))
 
+    #
     # Check AtomsBase Integration
+    #
     test_atom_objects = get_atom_objects(test_file)
     @test atomic_number(test_atom_objects[1]) == 7
     @test atomic_symbol(test_atom_objects[1]) == :N
     @test atomic_number(test_atom_objects[6]) == 6
     @test atomic_symbol(test_atom_objects[6]) == :C
 
+    #
+    # Check calculation methods
+    #
+
+    # populations, density, and mbo
+    test_calc_file = "./data/Trp_polar.fchk"
+    pop_funcs = [cspa, mpa, lpa, bpa]
+    for func in pop_funcs
+        aoresults, fragresults, fragcharges = func(test_calc_file)
+        @test size(aoresults) == (1, 87, 87)
+        @test typeof(aoresults) == Array{Float64, 3}
```

Comment:
Not sure whether using a for loop below is appropriate.
I'm assuming that not matter the input molecule, the sizes of all four population methods should remain consistent.

Reply:
For loop is good here, and yes, they should be consistent.